src/main/js/encounterDrawer.js

```diff
@@ -87,9 +88,36 @@ class EncounterDrawerUnstyled extends React.Component {
     this.refreshCombatantsState();
   }
 
-  updateCombatant(combatant) {
-    //TODO patch call
-    this.refreshCombatantsState();
+  updateCombatant(url, data) {
+      fetch(url, {
+          method: 'PATCH',
+          body: JSON.stringify(data),
+          headers: {
+            'Content-Type': 'application/json'
+          }
+      }).catch(err => err)
+      .then(() => this.refreshCombatantsState());
+  }
+
+  handleChange(combatantId, dataType, value) {
+    let data;
+    const newState = this.state.combatants.map(item => {
```

Comment:
I'm not sure I understand this logic are you calling the url with a patch like:

{
   currentHp: 32
}

Reply:
yes that is exactly whats happening. 
I am using your CompatnatApI
@PatchMapping("combatants/{combatantId}")
    public Combatant updateCombatant

Reply:
Okay just built and tested your branch on my computer.

Results:

I hit the API directly to test a modification to the currentHp: Used PostMan
localhost:8080/combatants/326
{
	"currentHp": 1
}

Successfully updated the hp to 1


I launched the UI and played with the text field.  Changing the text field to a new number and refreshing the page (via web browser)  kept the change.

Using math in the field didn't seem to have any effect and it just keep the hit points at whatever the number was.  example: +50 became 50